### PR TITLE
+htc #16972 makes bindAndStartHandlingWith actually run()

### LINKED ISSUE #16972: Http().bindAndStartHandlingWith does not work
there’s a `.run()` missing

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #16972

### CODE REVIEW

**Comment by rkuhn** [comment]
LGTM

File: akka-http-core/src/main/scala/akka/http/Http.scala [hunk 1]
```diff
@@ -60,13 +60,13 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
    * connections are being accepted at maximum rate, which, depending on the applications, might
    * present a DoS risk!
    */
-  def bindAndstartHandlingWith(handler: Flow[HttpRequest, HttpResponse, _],
+  def bindAndStartHandlingWith(handler: Flow[HttpRequest, HttpResponse, _],
```
Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/Http.scala]: there was also the question in #16972 on if this method should be called `bindAndHandle` to be more similar to TCP.
// cc @rkuhn @jrudolph @hseeberger @sirthias

Comment (hseeberger) [inline akka-http-core/src/main/scala/akka/http/Http.scala]: Please!

Comment (hseeberger) [inline akka-http-core/src/main/scala/akka/http/Http.scala]: `theseTerriblyLongNamesRemindMeOfMyJeeSpringDays`

Comment (rkuhn) [inline akka-http-core/src/main/scala/akka/http/Http.scala]: ah, right; yes, this makes sense.

Comment (ktoso) [inline akka-http-core/src/main/scala/akka/http/Http.scala]: Yeap, PR here https://github.com/akka/akka/pull/17042

**Pull request merged**